azure/template.json

```diff
@@ -0,0 +1,368 @@
+                    },
+                    "appInsightsResourceId": {
+                        "value": "[reference('ml-app-insights').outputs.AppInsightsResourceId.value]"
+                    },
+                    "containerRegistryResourceId": {
+                        "value": "[reference('ml-container-registry').outputs.acrResourceId.value]"
+                    },
+                    "confidentialData": {
+                        "value": "[parameters('confidentialData')]"
+                    }
+                }
+            },
+            "dependsOn": [
+                "[variables('resourceGroupName')]"
+            ]
+        },
+        {
+            "apiVersion": "2020-10-01",
+            "name": "ml-compute",
+            "type": "Microsoft.Resources/deployments",
+            "resourceGroup": "[variables('resourceGroupName')]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'),'machine-learning-services/machine-learning-compute.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "workspaceName": {
+                        "value": "[variables('workspaceName')]"
+                    },
+                    "computeName": {
+                        "value": "[variables('computeName')]"
+                    },
+                    "computeType": {
+                        "value": "AmlCompute"
+                    },
+                    "vnetResourceGroupName": {
+                        "value": "[parameters('sharedEnvResourceGroup')]"
+                    },
+                    "vnetName": {
+                        "value": "[parameters('sharedEnvVirtualNetworkName')]"
+                    },
+                    "subnetName": {
+                        "value": "[parameters('subnetObject').name]"
+                    }
+                }
+            },
+            "dependsOn": [
+                "[variables('resourceGroupName')]",
+                "ml-workspace"
+            ]
+        },
+        {
+            "apiVersion": "2020-10-01",
+            "name": "ml-datastore",
+            "type": "Microsoft.Resources/deployments",
+            "resourceGroup": "[variables('resourceGroupName')]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'),'machine-learning-services/machine-learning-datastore-blob.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "workspaceName": {
+                        "value": "[variables('workspaceName')]"
+                    },
+                    "datastoreName": {
+                        "value": "trainingdata"
```

Comment:
Could use `[variables('dataStorageContainerName')]` here if we're going to use the same name for the Storage Account Blob Container and the ML Data Store